doc/CONTRIBUTING.md

```diff
@@ -26,3 +26,8 @@ Copyright
 
 By contributing to this repository, you agree to license your work under the AGPL license.
 Please include the bisq license header into your files.
+
+Compensation for your contributions
+---------
+Bisq is not a company but operates as a [DAO](https://docs.bisq.network/dao/phase-zero.html). For any development work merged into the Bisq master branch you can file a [compensation request](https://github.com/bisq-network/compensation) and earn BSQ (the DAO native token). Learn more about the Bisq DAO at our [docs](https://docs.bisq.network/dao/phase-zero.html) pages.
+
```

Comment:
Just a comment unrelated to your changes for this PR, I was wondering if this CONTRIBUTING doc is necessary here? Should we merge all its content into the one under [bisq-docs](https://github.com/bisq-network/bisq-docs/blob/master/CONTRIBUTING.adoc)?
Also since that is where the main [CONTRIBUTING.md](https://github.com/bisq-network/bisq/blob/master/CONTRIBUTING.md) doc for this repo points to.

Reply:
I would like to leave it to @cbeams if he prefers to merge both files.

Reply:
`CONTRIBUTING.md` is there because it is recognized and dealt with specially by GitHub. Namely, it asks users to read the file when submitting a PR (as you may have noticed). So we could make it a placeholder with link(s) pointing to canonical doc(s) elsewhere in the repo, but the file itself should remain in place, as it's the hook into GitHub's PR system.